CHANGELOG.md

```diff
@@ -8,13 +8,24 @@ Links "DE#nnn" prior to version 2.0 point to the Dash Enterprise closed-source D
 
 ### Changed
```

Comment:
There should be a new version number added rather than `unreleased` similar to others

Reply:
unreleased swaps to version number with the release of the new package, as we dont know what version these changes will be included in when adding. 😏

Reply:
This is the release PR though?